.config/TSAOptions.json

```diff
@@ -0,0 +1,12 @@
+{
+    "tsaVersion": "TsaV2",
+    "codebaseName": "NuGet.Client-Trusted_dev",
+    "notificationAliases": [
+        "NuGetHotSeat@service.microsoft.com"
```

Comment:
It's too late now, but I would really prefer not to have our email address listed in a public location, to reduce risk of spam from people who scrape email addresses.  If there's no alternative, we need to report this to the Code QL team because it's just not appropriate for OSS repos on github.